### PR TITLE
kafka

### LINKED ISSUE #10: kafka impl

### PR DESCRIPTION
**Pull request by eyakauleva**
close #10

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -14,6 +21,18 @@ spring:
     default-property-inclusion: non_null
   application:
     name: TICKET-SERVICE
+  kafka:
+    topic: users
+    bootstrap-servers: localhost:9092
```
Comment (h1alexbel) [inline src/main/resources/application.yaml]: host is hardcoded

File: src/main/java/com/solvd/micro9/tickets/messaging/KfConsumer.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class KfConsumer {
+
+    private final ReactiveKafkaConsumerTemplate<String, Long> receiver;
+    private final TicketService ticketService;
+
+    @PostConstruct
+    public void fetch() {
```
Comment (h1alexbel) [inline src/main/java/com/solvd/micro9/tickets/messaging/KfConsumer.java:19]: try to declare methods only in interfaces

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Comment by h1alexbel** [comment]
@eyakauleva fix kafka host please

File: src/main/resources/application.yaml (hunk 1 above)
Comment (eyakauleva) [reply src/main/resources/application.yaml]: fix in 34ee808

File: src/main/java/com/solvd/micro9/tickets/messaging/KfConsumer.java (hunk 2 above)
Comment (eyakauleva) [reply src/main/java/com/solvd/micro9/tickets/messaging/KfConsumer.java:19]: you mean creating KfConsumer interface and then its implementation?

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**